### PR TITLE
fix(philosophy): enable fluid type mixin

### LINKED ISSUE #136: Philosophy page: Body copy needs to be Expressive H3 I believe
@jeanservaas can confirm size name just in case I’m wrong.

current:

Screen Shot 2019-07-31 at 12 51 00 PM

——————

expected:

Screen Shot 2019-07-31 at 12 50 54 PM

### PR DESCRIPTION
**Pull request by emyarod**
Closes #136

This PR enables the `fluid` option for the type mixin for body copy on the Philosophy page

### CODE REVIEW

**Comment by jeanservaas** [comment]
Just noticed that blue type needs to be the same size and line height as the $expressive-heading-03... it's just blue and italic

And then this is 14px Plex Sans Italic

_– Paul Rand_

(ps that's an en-dash tee hee))

image

**Comment by emyarod** [comment]
should be resolved now @jeanservaas

**Comment by alisonjoseph** [comment]
@emyarod looks like we lost the padding to the right of the blockquote by switching it to html from markdown. This might be an update we need to make in the theme here I think https://github.com/carbon-design-system/gatsby-theme-carbon/blob/master/packages/gatsby-theme-carbon/src/components/markdown/Markdown.module.scss#L41

**Comment by alisonjoseph** [comment]
opened a PR https://github.com/carbon-design-system/gatsby-theme-carbon/pull/313

File: src/pages/philosophy/pov.mdx [hunk 1]
```diff
@@ -29,8 +29,13 @@ Paradoxically, decisions are more distributed, and experiences can become less a
 
 It is more important than ever that we own our own ethos, make palpable our brand values, and incorporate an instantly identifiable IBMness in everything we do.
 
-> Without aesthetic, design is either the humdrum repetition of familiar clichés or a wild scramble for novelty. Without aesthetic, the computer is but a mindless speed machine, producing effects without substance, form without relevant content, or content without meaningful form.
-> – Paul Rand
+<blockquote>
```
Comment (alisonjoseph) [inline src/pages/philosophy/pov.mdx]: Can we revert these changes back? Fix in the theme should fix it and still allow for us to use the markdown formatting

Comment (emyarod) [reply src/pages/philosophy/pov.mdx]: sure, done

**Comment by emyarod** [comment]
@alisonjoseph does that mean we can close this PR and push all the changes to the theme side?

**Comment by alisonjoseph** [comment]
I think we can still merge this PR as it for the large type, since that is a site-specific update. and then merge in the fix for the blockquote, since that change is theme-wide.

**Comment by mjabbink** [comment]
also looks like the sidebar is more narrow in the gatsby IDL versa the current. Is the grid column narrower or margin?

**Comment by mjabbink** [comment]
Not seeing tis fixed in staging.

**Comment by alisonjoseph** [comment]
@mjabbink the grid column is correct, the issue is coming from the default extra right spacing we have added to headings / paragraphs. I'll fix. 

The blockquote text was a fix that we made in the theme and just released and hasn't been pulled back in to the staging site yet. 

The paragraph text is using the expressive heading token as far as I can tell

**Comment by mjabbink** [comment]
we’re talking blue quote right? Should be same as body copy size just blue and italic.

**Comment by alisonjoseph** [comment]
Yes, that will be fixed once this PR is merged in https://github.com/carbon-design-system/design-language-website/pull/159

you can see it working here in the preview link https://deploy-preview-159--design-language-website.netlify.com/philosophy/pov/

**Comment by mjabbink** [comment]
yes, but in mobile the text should go to right edge of the screen and the skinny sidebar text with rule is bleeding on the left edge.


IMG_3854

**Comment by mjabbink** [comment]
IMG_3855

**Comment by mjabbink** [comment]
there are some strange breaks in the body as well that show up in mobile so some lines go to the right edge and some not when they have room too

**Comment by mjabbink** [comment]
size is correct now though

**Comment by alisonjoseph** [comment]
Will tackles all of that in this issue 👍 definitely something weird happening there. 
https://github.com/carbon-design-system/design-language-website/issues/157
